.github/workflows/ci.yml

```diff
@@ -29,23 +29,20 @@ jobs:
   release:
     if: github.ref_type == 'tag'
     needs: build
-    runs-on: ubuntu-20.04
```

Comment:
I suggest we bump this to latest which I believe is 22.04 to avoid using generic runners which might in the future following OS updates break this as this won't have been tested.